array/DNAm/preprocessing/jobSubmission/1_runDNAmQC.sh

```diff
@@ -34,6 +34,8 @@ module load $RVERS   # load specified R version
 
 cd ${SCRIPTSDIR}/array/DNAm/preprocessing/
 
+Rscript checkRconfigFile.r ${DATADIR}
```

Comment:
I haven't read through this file properly yet, but it looks as though if the config file is malformed, the following commands in the jobsubmission shell script will run anyways.

Although the addition of the config checker is a great step, it would be great if execution of the entire shell script stops in such cases (as otherwise, the user might not know about these errors if they don't check the logs)

Reply:
I have an idea of how to do this, which I will try to implement tomorrow.

Reply:
done.

Reply:
It would be nice to add this after the checkColnamesSampleSheet.r script too

Reply:
I've created a issue for this